5.sc_Morphology_Logistic_Regression/README.md

```diff
@@ -0,0 +1,13 @@
+
+All code in this directory is inspired or taken from the following repository: https://github.com/WayScience/phenotypic_profiling_model/tree/main
```

Comment:
Thanks for citing this ❤️. I guess this means you didn't have to struggle very long with catching the warnings like I mentioned earlier haha.